src/main/java/core/basesyntax/service/operations/SupplyOperationsHandler.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax.service.operations;
+
+import core.basesyntax.model.Operations;
+
+public class SupplyOperationsHandler implements OperationsHandler {
```

Comment:
There is an unnecessary empty line at the beginning of the class definition. Please remove it to adhere to the coding standards.